conda/base/context.py

```diff
@@ -1256,6 +1256,22 @@ def auto_activate_base(self) -> bool:
     def default_activation_env(self) -> str:
         return self._default_activation_env or ROOT_ENV_NAME
 
+    @property
+    def default_activation_prefix(self) -> PathType:
```

Comment:
Let's always return a `pathlib.Path` here:

```suggestion
    def default_activation_prefix(self) -> Path:
```

Reply:
Okay, I've casted the return value of `context.default_activation_prefix` to `Path` as well.